lms/djangoapps/instructor_task/tests/test_tasks_helper.py

```diff
@@ -869,60 +869,6 @@ def test_cohort_content(self):
         expected_grades = [self._format_user_grade(header_row, **user_grade) for user_grade in user_grades]
         self.verify_rows_in_csv(expected_grades)
 
-    @patch('courseware.grades.MaxScoresCache.get', Mock(return_value=1))
-    def test_cohort_content_with_maxcache(self):
-        """
-        Tests the cohoted course grading to test the scenario in which `max_scores_cache` is set for the course
-        problems.
-        """
```

Comment:
This test will remain deleted - the above test checks the same functionality, this was guarding against odd behavior from max_scores_cache, which no longer exists.